source/blog/_posts/2016-09-09-99-paper-cuts-status.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Status of the 99 papercuts project
+github_link: blog/_posts/2016-09-09-99-paper-cuts-status.md
+authors: [bc]
+---
+
+We are at day five of our papercuts project, ten days to go. 
+
+Continuing until Monday, 19th September you can participate in our [99 Papercuts Project](https://developers.shopware.com/99-paper-cuts/) and make Shopware better - one papercut at a time.
```

Comment:
From now until Monday, 19th September, you can participate in our [99 Papercuts Project](https://developers.shopware.com/99-paper-cuts/) and make Shopware better - one papercut at a time.